packages/admin-ui-docs/docs/components/checkbox.mdx

```diff
@@ -511,31 +532,157 @@ function Example() {
   direction="row"
 >
 
-- Checkbox has role `checkbox`.
+- Checkbox has the `checkbox` role.
 - When checked, Checkbox has `aria-checked` set to `true`.
-- When not checked, Checkbox has `aria-checked` set to `false`.
-- When partially checked, Checkbox has `aria-checked` set to `mixed`.
+- When unchecked, Checkbox has `aria-checked` set to `false`.
+- When indeterminate, Checkbox has `aria-checked` set to `mixed`.
 
 </PageSection>
 
   </Panel>
     <Panel id="designReference">
 
-## Do’s
-
-1. Settings confirmation, that require an additional action to activate.
-2. Multiple choices.
-3. Undetermined state, when multiple sub-options are grouped under a parent option.
-4. Clear visual state for selected and unselected options.
-5. List of related items.
-6. Binary options are available, like yes/no choices.
-7. Use titles to group related controls.
-8. Choose words that are aligned with our ecommerce vocabulary.
-9. Write the current state of the control, to improve accessibility.
-10. Add descriptions if the control is critical, or complex.
-11. Lists of controls should be vertical for better scannability
-
-👉 For UX Writing guidelines, access https://uxwriting.vtex.com
+<PageSection
+  id={tabToc.designReference[0].id}
+  title={tabToc.designReference[0].value}
+>
+
+<details><summary>When to use the Radio, Switch, or Combobox Multiple components instead of the Checkbox component?</summary>
+<p>
+  <ul>
+    <li>Use the Checkbox component to allow multiple choices, in scenarios where options are not mutually exclusive and you wish to allow one or more selection.</li>
+    <li>Use the Checkbox component to provide binary options, such as yes or no choices, where checking the Checkbox means yes and not checking it means no. Never use a radio component in this scenario.</li>
+    <li>Choose the Combobox Multiple component if multiple options can be selected and there are more than 5 available options.</li>
+    <li>Choose the Switch component if the action is instant and does not require confirmation. For example, use a switch in a product form to show a new form section only if the product has variants.</li>
+    <li>Choose the Radio component if only one option can be selected at a time.</li>
```

Comment:
Choose the Radio component if only one option can be selected at a time and there are up to five options.